framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Pagination/PagerModel.cs

```diff
@@ -38,6 +38,11 @@ public PagerModel(long totalCount, int shownItemsCount, int currentPage, int pag
             PageSize = pageSize;
             TotalPageCount = (int)Math.Ceiling(Convert.ToDouble((decimal)TotalItemsCount / PageSize));
             Sort = sort;
+
+            if (pageUrl.IsNullOrWhiteSpace() || !pageUrl.StartsWith("/"))
```

Comment:
You can use `pageUrl = pageUrl?.EnsureStartsWith('/') ?? "/"`.

See https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Core/System/AbpStringExtensions.cs#L33

Reply:
Ok